src/vma/hw/mlx5/wqe.h

```diff
@@ -33,7 +33,12 @@
 #ifndef WQE_H
 #define WQE_H
 
-#define MLX5_ETH_INLINE_HEADER_SIZE 16
+#if !defined(MLX5_ETH_INLINE_HEADER_SIZE)
+#define MLX5_ETH_INLINE_HEADER_SIZE 18
+#endif
```

Comment:
@olgk this code is useless because `MLX5_ETH_INLINE_HEADER_SIZE ` is `enum value`.  So `#if !defined(MLX5_ETH_INLINE_HEADER_SIZE)` does not work in this case.